electrum/transaction.py

```diff
@@ -1482,7 +1483,11 @@ def convert_raw_tx_to_hex(raw: Union[str, bytes]) -> str:
     if not raw:
         raise ValueError("empty string")
     raw_unstripped = raw
-    raw = raw.strip()
+    # remove all whitespace characters
+    if isinstance(raw, str):
+        raw = re.sub(r'\s', '', raw)
+    else:
+        raw = re.sub(rb'\s', b'', raw)
```

Comment:
it is not safe to remove bytes that look like whitespace characters from a bytes-like raw tx input.
However note we were already doing this for leading/trailing bytes. That is not safe either. E.g. the last byte of the nLocktime might look like a whitespace character - it should not be stripped.
I added follow-up commits to address this.